app/page.tsx

```diff
@@ -1,17 +1,16 @@
 import WelcomeTexts from '@/components/welcome-texts/WelcomeTexts';
+import { PostsFixLoader } from '@/components/posts-loader/PostsFixLoader';
 
-import PostsLoader from '@/components/posts-loader/PostsLoader';
-
-export default async function Page() {
+export default function Page() {
   return (
-    <div className="bg-slate-100 pt-8">
-      <div className="intro global-width mx-auto py-8">
+    <div className="flex flex-col overflow-hidden">
+      <div className="global-width mx-auto py-2 pt-8">
         <WelcomeTexts
           title="Welcome to Mumble"
-          description="Did you hear that? They've shut down the main reactor."
+          description="Login to have a full experience!"
```

Comment:
This page should be only reachable if the user is not loggedIn. Otherwise we need to redirect to the Custom Feed. 
I will tackle this improvement as a bug.